lib/internal/cluster/child.js

```diff
@@ -128,7 +128,7 @@ function rr(message, indexesKey, cb) {
   if (message.errno)
     return cb(message.errno, null);
 
-  var key = message.key;
+  const key = message.key;
```

Comment:
```suggestion
  let key = message.key;
```
The `key` is later assigned `undefined` in the `close()` below, so it should use `let` here.